client/client.go

```diff
@@ -111,7 +111,7 @@ func (c *DatabricksClient) Do(ctx context.Context, method, path string,
 	if strings.HasPrefix(path, "/api/2.0/fs/files//") {
 		path = strings.Replace(path, "/api/2.0/fs/files//", "/api/2.0/fs/files/", 1)
 	}
```

Comment:
Unrelated: with the recent updates to the Files API spec, can we remove this?

Reply:
Yes, we'll do that as a fast follow.